.github/workflows/connector-tests.yml

```
@@ -101,17 +101,49 @@ jobs:
+      # Upload the job output to the artifacts
+      - name: Upload Job Output
+        id: upload_job_output
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{matrix.connector}}-job-output
+          path: airbyte/airbyte-ci/connectors/pipelines/pipeline_reports
+
+      - name: Evaluate Job Output
+        if: always()
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:18: Double quote to prevent globbing and word splitting

(shellcheck)